src/tests/snapshot.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { StaticRouter } from 'react-router-dom';
+import Header from '../components/Header';
+// import * as styles from '../components/Header.css';
```

Comment:
- [ ]  Kindly remove any unused commented code, this will make your code cleaner